docs/src/man/getting_started.md

```diff
@@ -116,6 +116,27 @@ Note that constructing a `DataFrame` row by row is significantly less performant
 constructing it all at once, or column by column. For many use-cases this will not matter,
 but for very large `DataFrame`s  this may be a consideration.
 
+### Constructing from another table type
+
+DataFrames supports the [Tables.jl](https://github.com/JuliaData/Tables.jl) interface for
+interacting with tabular data. This means that a DataFrame can be used as a "source"
+to any package that expects a Tables.jl interface input, (file format packages,
+data manipulation packages, etc.). A DataFrame can also be a sink for any Tables.jl
+interface input. Some example uses are:
+
+```julia
+df = DataFrame(a=[1, 2, 3], b=[:a, :b, :c])
+
+# write DataFrame out to csv file
+df |> CSV.write("dataframe.csv")
```

Comment:
Should add `using CSV, SQLite, Query`.

BTW, in the paragraph above, no comma before parenthesis.

Reply:
Please don't advocate piping for such a simple operation in the documentation...